avaframe/tests/test_cfgUtils.py

```diff
@@ -52,3 +55,83 @@ def test_cfgHash():
     uid = cfgUtils.cfgHash(cfg)
     # make sure it is not the same hash
     assert uid != 'bcc6c69699'
+
+
+def test_convertConfigParserToDict():
+    """ test conversion of cfg to dict """
+
+    cfg = configparser.ConfigParser()
+    cfg['GENERAL'] = {'name': 'nameTest', 'relTh': '1.0'}
+    cfg['TEST1'] = {'mu': '0.15500', 'id': 'name'}
+
+    cfgDict = cfgUtils.convertConfigParserToDict(cfg)
+
+    assert cfg['GENERAL'] == cfgDict['GENERAL']
+    assert cfg['TEST1'] == cfgDict['TEST1']
+
+
+def test_convertDictToConfigParser():
+    """ test conversion to cfg """
+
+    cfgDict = {}
+    cfgDict['GENERAL'] = {'name': 'nameTest', 'relTh': 1.0}
+    cfgDict['TEST1'] = {'mu': 0.15500, 'id': 'name'}
+
+    cfg = cfgUtils.convertDictToConfigParser(cfgDict)
+
+    assert cfgDict['GENERAL']['relTh'] == cfg['GENERAL'].getfloat('relTh')
+    assert cfgDict['TEST1']['mu'] != cfg['TEST1']['mu']
+    assert cfgDict['TEST1']['mu'] == cfg['TEST1'].getfloat('mu')
+
+
+def test_orderSimFiles():
+    """ test generating order of simulation results """
+
+    avaTestDir = 'avaHockeyChannelPytest'
+    dirPath = os.path.dirname(__file__)
+    avaDir = os.path.join(dirPath, '..', '..', 'benchmarks', avaTestDir)
+    inputDir = os.path.join(avaDir, 'Outputs', 'com1DFA', 'peakFiles')
```

Comment:
I suggest switching all the os.path.joins to pathlib functions (also further down)

Reply:
now all paths in here using pathlib